src/screens/Wallet/Send/Send.js

```diff
@@ -40,11 +126,80 @@ export default class Send extends PureComponent {
           sendConfirm={onSendConfirm}
         />
         }
+        <View style={styles.formHeader}>
+          <Text style={styles.walletTitle}>
+            {
+              strings.walletTitle
+            }
+          </Text>
+          <Text style={styles.walletAddress}>
+            {
+              selectedWallet.address
+            }
+          </Text>
+          {
+            (blockchain === DUCK_ETHEREUM)
+              ? (
+                <View>
+                  <Separator style={styles.separatorDark} />
+                  <TokenSelector
+                    selectedToken={selectedToken}
+                    onPress={onSelectToken}
+                  />
+                </View>
+              ) : null
+          }
+          <Separator style={styles.separatorDark} />
+          <Text style={styles.walletValue}>
+            {
+              strings.walletValue
+            }
+          </Text>
+          <Text style={styles.walletBalance}>
+            {
+              strings.walletBalance
+            }
+          </Text>
+        </View>
+        <View style={styles.formBody}>
+          <Image
+            source={cryptoImages[blockchain] || coin_time_small}
+            style={styles.tokenImage}
+          />
+          <Input
+            placeholder='Recipient Address'
+            onChange={onChangeRecipient}
+            name='recipient'
+          />
+          <Input
+            placeholder={strings.amountInput}
+            keyboardType='numeric'
+            onChange={onChangeAmount}
+            name='amount'
+          />
+          <Text style={styles.sendBalance}>
+            {
+              strings.sendBalance
+            }
+          </Text>
+          <FeeSlider
+            tokenSymbol={selectedToken.symbol}
+            selectedCurrency={selectedCurrency}
+            calculatedFeeValue={gasFeeAmount}
+            calculatedFeeValueInSelectedCurrency={gasFeeAmountInCurrency}
+            maximumValue={1.9}
+            minimumValue={0.1}
+            value={feeMultiplier}
+            step={0.1}
+            handleValueChange={onFeeSliderChange}
+          />
+        </View>
         <PrimaryButton
           label='Go to password entrance'
           onPress={onTogglePasswordModal}
+          style={styles.openModalButton}
```

Comment:
This button must be in the header. See https://test.chronobank.io/iOS/send.html
Its title should be "Done"